### PR TITLE
WIP: TFS 1114543: added event attribute and note about interaction with web-source

### PR DESCRIPTION
**Pull request by dzucker-tab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,19 +96,14 @@ Content of details.xml:
 ```
 <tsRequest>
 
-  <webhook name="my_first_webhook">
+  <webhook name="my_first_webhook"   
+    event="webhook-source-event-datasource-created" >
```
Comment (shinchris) [inline README.md:100]: `event` values are not the same as what `webhook-source` used to have. This line should be `event="DatasourceCreated" now.

File: README.md [hunk 2]
```diff
@@ -247,16 +242,22 @@ POST /api/3.6/sites/<site-id>/webhooks
...
-  <webhook name="webhook-name" isEnabled="webhook-enabled-flag" >  
-    <webhook-source>  
-      <webhook-source-api-event-name />  
-    </webhook-source>  
-    <webhook-destination>  
-      <webhook-destination-http method="POST" url="url" />  
-    </webhook-destination>
-  </webhook>  
-</tsRequest>
+<tsResponse>
```
Comment (shinchris) [inline README.md:245]: The title says request, but the payload says response. Owner tag should also be removed

Comment (dzucker-tab) [reply README.md:247]: fixed in latest commit

Comment (dzucker-tab) [reply README.md]: fixed in latest commit

File: README.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ using Postman.
 
 1. Open the **MANAGE ENVIRONMENTS** dialog from the gear icon, open **Tableau Webhooks** variables and use the site id and token to set **CURRENT VALUE** of the `site_id` and `tableau_auth_token` variables.
 
-1. In the list of requests, click **Create a webhook**. In the request body, enter a webhook `name`, a  `webhook-source-api-event-name` from the [Events](#events) table, and a destination `url`. The destination URL must be https and have a valid certificate.
+1. In the list of requests, click **Create a webhook**. In the request body, enter a webhook `name`, a Tableau trigger event () from the [Trigger Events](#events) table, and a destination `url`. The destination URL must be https and have a valid certificate.
```
Comment (shinchris) [inline README.md:59]: Open parenthesis in the second sentence

File: README.md [hunk 4]
```diff
@@ -473,7 +499,17 @@ PUT /api/3.8/sites/<site-id>/webhooks/<webhook-id>
 
 `webhook-name`   Required. A name for the webhook.
 
-`webhook-source-api-event-name`   Required. The API event name for the source event. It must be one of the supported events, such as, \<webhook-source-event-datasource-refresh-started />  
+
+`api-event-name`  | `webhook-source-api-event-name`  
+
+ Optional. The name of the Tableau event that triggers your webhook. The event name  must be one of the supported events listed in the [Triggering Events](#events) table. Note that `event` and `webhook-source` use different name values for the same event.
```
Comment (shinchris) [inline README.md:505]: I think this one should also say "One of the other is required", like it is for the create endpoint.

Comment (jzsheeran) [reply README.md]: Took a slightly different approach because, AFAIK, specifying event for an update is optional, no?

File: README.md (hunk 3 above)
Comment (dzucker-tab) [reply README.md]: fixed in latest commit

File: README.md (hunk 4 above)
Comment (dzucker-tab) [reply README.md]: Took a slightly different approach to account for event being optional for an update. Is that not right?

Comment (shinchris) [reply README.md]: Ah yes, that is right.

**Pull request merged**